core/src/main/java/gov/nasa/jpl/mudrod/ssearch/ClickstreamImporter.java

```diff
@@ -63,7 +65,7 @@ public void addClickStreamMapping() {
    * Method to import click stream CSV into Elasticsearch
    */
   public void importfromCSVtoES() {
-    es.deleteType(props.getProperty("indexName"), props.getProperty("clickstreamMatrixType"));
+    es.deleteType(props.getProperty("indexName"), props.getProperty(MudrodConstants.CLICK_STREAM_MATRIX_TYPE));
```

Comment:
we should change props.getProperty("indexName") to props.getProperty(MudrodConstants.INDEX_NAME) in this line